docs/vmalert-tool.md

```diff
@@ -33,6 +33,28 @@ except `promql_expr_test` field. Use `metricsql_expr_test` field name instead. T
 validates and executes [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html) expressions,
 which aren't always backward compatible with [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/).
 
+### Limitations
+
+* vmalert-tool evaluates all the groups defined in `rule_files` using `evaluation_interval`[default 1m] instead of `interval` under each rule group.
+* vmalert-tool shares the same limitation with [vmalert](https://docs.victoriametrics.com/vmalert.html#limitations) on chaining rules under one group:
```

Comment:
what would be the implications if rules within a group would be executed on the full range? In the similar way how vmalert implements `replay`: each rule within the group evaluated on the full range and results are flushed to the database. This would allow chaining...
On the other hand, this is synthetic and in the runtime there are no guarantees for chaining to be respected.

Reply:
>if rules within a group would be executed on the full range

@hagen1778  I think we shouldn't do that, vmalert-tool test is meant to test rules, and if test cases are successful, rules will be applied to vmalert, so we must keep the same logic between vmalert-tool and vmalert. 
>there are no guarantees for chaining to be respected.

Since we don't recommend users to having chaining rules under one group for non-guaranteed reasons, we shouldn't encourage them to do it in vmalert-tool by letting those tests through.

I think replay mode is special here, it gives the most ideal results)